app/uk/gov/hmrc/eoricommoncomponent/frontend/models/address/AddressLookup.scala

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.eoricommoncomponent.frontend.models.address
+
+import play.api.libs.json._
+import play.api.libs.json.Reads._
+import play.api.libs.functional.syntax._
+
+case class AddressLookup(addressLine: String, city: String, postcode: String, country: String)
+
+object AddressLookup {
+
+  def applyWithLines(lines: Seq[String], town: String, postcode: String, country: String): AddressLookup = {
+    val addressLine = lines match {
+      case Seq(line1, line2, _ @_*) => line1 + " " + line2
+      case Seq(line1, _ @_*)        => line1
```

Comment:
Probably worth to add case when there is no address lines. Shouldn't happen, but just to cover the scenario.

Reply:
Done